python/lsst/ts/scheduler/driver/feature_scheduler.py

```diff
@@ -0,0 +1,443 @@
+        survey_topology.sequence_propos = []
+
+        return survey_topology
+
+    def cold_start(self, observations):
+        """Rebuilds the internal state of the scheduler from a list of
+        observations.
+
+        Parameters
+        ----------
+        observations : list of Observation objects
+
+        """
+        raise RuntimeError("Cold start not supported by SequentialScheduler.")
+
+    def update_conditions(self):
+        """Update conditions on the scheduler.
+        """
+
+        super().update_conditions()
+
+        self._format_conditions()
+
+        # Update conditions on the scheduler
+        self.scheduler.update_conditions(self.conditions)
+
+        # Set time from next observation based on the current time on the
+        # observatory model, which accounts for current observations on the
+        # queue.
+        self.next_observation_mjd = self.models["observatory_model"].dateprofile.mjd
+
+    def select_next_target(self):
+        """Picks a target and returns it as a target object.
+
+        Returns
+        -------
+        Target
+
+        """
+
+        if self.next_observation_mjd is None:
+            raise RuntimeError(
+                "Time for next observation not set. Call `update_conditions` before requesting a target."
+            )
+
+        desired_obs = self.scheduler.request_observation(mjd=self.next_observation_mjd)
+
+        if desired_obs is not None:
+            target = FeatureSchedulerTarget(desired_obs)
+
+            slew_time, error = self.models["observatory_model"].get_slew_delay(target)
+
+            if error > 0:
+                self.log.error(
+                    f"Error[{error}]: Cannot slew to target @ ra={target.ra}, dec={target.dec}."
+                )
+                self.scheduler.flush_queue()
+                return None
+            else:
+                target.slewtime = slew_time
+
+                target.observation["mjd"] = (
+                    self.conditions.mjd + slew_time / 60.0 / 60.0 / 24.0
+                )
+                target.observation["night"] = self.conditions.night
+                target.observation["slewtime"] = slew_time
+
+                hpid = _raDec2Hpid(self.nside, target.ra_rad, target.dec_rad)
+
+                target.observation["skybrightness"] = self.conditions.skybrightness[
+                    target.filter
+                ][hpid]
+                target.observation["FWHMeff"] = self.conditions.FWHMeff[target.filter][
+                    hpid
+                ]
+                target.observation["airmass"] = self.conditions.airmass[hpid]
+                # target.observation["fivesigmadepth"] = m5_flat_sed(
+                #     filtername,
+                #     target.observation["skybrightness"],
+                #     target.observation["FWHMeff"],
+                #     target.observation["exptime"],
+                #     target.observation["airmass"],
+                # )
```

Comment:
Nitpicking again but please remove commented out code or add a comment explaining why it has to stay. Same for lines 276 and 277.